src/constants/constants.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export const TRANSPORTATION = 'Transportation'
+export const FOOD = 'Food'
+export const LODGING = 'Lodging'
+export const SIGHTSEEING = 'Sightseeing'
+export const OTHER = 'Other'
+export const ENTERTAINMENT = 'Entertainment'
+export const FUN_ACTIVITIES = 'Fun Activities'
+export const LIST_OF_EXPENSE_CATEGORIES = [TRANSPORTATION, FOOD, LODGING, SIGHTSEEING, OTHER]
+export const LIST_OF_PLACES_CATEGORIES = [SIGHTSEEING, FOOD, ENTERTAINMENT, FUN_ACTIVITIES, OTHER]
+
+export const DEFAULT_MAP_OPTIONS = {
```

Comment:
Imho takie coś pasuje przy komponencie, który z tego korzysta niż w globalnych consts.

Reply:
ale obecnie korzystaja z tego dwa componenty i beda korzystac kolejne, wiec chyba lepiej w constach?

Reply:
meh no nie wiem. zależy co wyjdzie z tego co napisałem wyżej z `RawMap`. consts to już taki bardzo globalny stan a jednak default map options są czymś powiązane bezpośrednio z komponentem mapki. Imho to pokazuje, że brakuje właśnie takiego low level'owego komponentu mapki. Bo to nie jest coś z czego korzystasz w searchu, mapach i jakimś menu.